src/DependencyInjection/CmfSonataAdminIntegrationExtension.php

```diff
@@ -68,4 +89,22 @@ public function getXsdValidationBasePath()
     {
         return __DIR__.'/../Resources/config/schema';
     }
+
+    private function defineFactories(ContainerBuilder $container)
```

Comment:
as there is no stable release, i propose we drop this method and just set up factories from the bundle class